api/types/common/constants.go

```diff
@@ -69,6 +69,11 @@ const (
 	// OriginEntraID indicates that the resource was imported
 	// from the Entra ID directory.
 	OriginEntraID = "entra-id"
+
+	// OriginAWSIdentityCenter indicates that the resource was
+	// imported from the AWS Identity Center or belongs to the
+	// Teleort Identity Center integration.
+	OriginAWSIdentityCenter = "aws-identity-center"
```

Comment:
@tcsc FYI, I added this origin label with `aws-identity-center` here. In the poc work, we used `aws_iam_ic`. Is there any preference for you? My reason for picking up `aws-identity-center` is that is used in the Plugin related work but should not hamper if this is different.

Reply:
`OriginAWSIdentityCenter` is removed from this PR now and I will add this directly to the main plugin backend implementation PR.